src/tests/specs/agileTemplate.spec.ts

```diff
@@ -56,4 +56,16 @@ describe('Agile template tests: ', () => {
     await planner1.quickPreview.businessValue.untilTextIsPresentInValue('Business value for this Theme');
     expect(await planner1.quickPreview.businessValue.getAttribute('value')).toBe('Business value for this Theme');
   });
+
+  it('Dynamic fields should not get closed on outside click', async () => {
+    let newWorkItem = { title: 'Workitem of type Story', type : 'Storypoints'};
+    await planner1.createWorkItem(newWorkItem);
+    expect(planner1.workItemList.hasWorkItem(newWorkItem.title)).toBeTruthy();
+    /* Edit Story Points */
+    await planner1.workItemList.clickWorkItem(newWorkItem.title);
+    await planner1.quickPreview.storyPoints.clickWhenReady();
+    expect(await planner.quickPreview.isSaveButtonDisplayed()).toBeTruthy();
+    await planner1.quickPreview.labelDropdown.clickWhenReady();
+    expect(await planner.quickPreview.isSaveButtonDisplayed()).toBeTruthy();
```

Comment:
Instead of the Save button, a better check would be to evaluate the value in the field.